{{ cookiecutter.project_slug }}/{{ cookiecutter.package_name }}/strategies/parse.py

```diff
@@ -52,7 +51,7 @@ def parse(self, session: "Optional[Dict[str, Any]]" = None) -> "Dict[str, Any]":
             dictionary context.
 
         """
-        downloader = create_download_strategy(self.resource_config)
+        downloader = create_strategy("download", self.resource_config)
```

Comment:
self.resource_config must be replaced by self.parse_config according to your change in line 20.